framework/include/cppmicroservices/BundleContext.h

```diff
@@ -443,7 +443,7 @@ namespace cppmicroservices
          *        services.
          * @return A list of <code>ServiceReference</code> objects or
          *         an empty list if no services are registered which satisfy the
-         *         search.
+         *         search. These objects will be in decreasing order of their rank.
```

Comment:
this should be added to the documentation for `GetServiceReferences` on line 428.

Reply:
@jeffdiclemente fixed